### PR TITLE
Readme review 28619

### PR DESCRIPTION
**Pull request by fierytortilla**
Let me know how you feel about my reviews thus far and what you might want me to change.

### CODE REVIEW

**Comment by simon-tarr** [comment]
Is this pull request still required @fierytortilla?

**Comment by fierytortilla** [comment]
@simon-tarr I'm not really sure if it is required anymore, since so many other things has happened since I last did this pull request. I would prefer if we could discuss my proposals altogether as a team in the next Code review/retro so that we can have a good holistic conversation about what we want/don't want.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,49 +1,52 @@
...
-If you reference a work item by id in a commit message - e.g. #12345 
-Fixed issue where data was eaten by a grue - DevOps will auto link the item 
+If you reference a work item by id in a commit message - e.g. #12345
+Fixed issue where data was eaten by a grue - DevOps will auto link the item
 to the commit, and therefore the PR.
 
-Otherwise you can manually link items at the point of PR creation, or 
+Otherwise you can manually link items at the point of PR creation, or
 afterwards, if necessary.
+#DS- Should there be more details added to the branch naming scheme other than the sprint work item number?
```
Comment (py5gol) [inline CONTRIBUTING.md:26]: The proposal is <ticket_id>-<brief description>, i.e. 29973-docker-configuration.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,49 +1,52 @@
...
+#DS- How will the Azure Devops work items and Git by manually linked afterwards if a commit message is not referenced?
```
Comment (py5gol) [inline CONTRIBUTING.md:27]: See example above for commit messages. Branch names do not take the symbol #.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,49 +1,52 @@
...
+trivial - if in doubt, err on the side of more documentation. See
 https://www.python.org/dev/peps/pep-0257/ for full guidance on writing docstrings.
 Note the use of triple-double quotes (""") to start and end docstrings.  
 
 ### Testing
-All PRs should also contain tests that test the functionality within 
-the PR. 
+All PRs should also contain tests that test the functionality within
+the PR.
+#DS- What kind of testing package or testing framework should be utilized (e.g. unittest, pytest, robot)? This should be specified.
```
Comment (py5gol) [inline CONTRIBUTING.md:46]: If writing pure Python code, pytest. If Django code, then Django's testing library. If .NET, then xunit.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,49 +1,52 @@
...
 
 ### Continuous integration
-In the near future, Continuous Integration tests will run on each PR, to 
-ensure that all tests pass before the PR can be merged into `master`. 
+In the near future, Continuous Integration tests will run on each PR, to
+ensure that all tests pass before the PR can be merged into `master`.
 More details will be available when this is in place.
-
+#DS- More information should be made available for new developers not familiar with CI techniques.
```
Comment (py5gol) [inline CONTRIBUTING.md:52]: Yes, but we will add this info later.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
-# Introduction 
-TODO: Give a short introduction of your project. Let this section explain the objectives or the motivation behind this project. 
+# Introduction
+TODO: Give a short introduction of your project. Let this section explain the objectives or the motivation behind this project.
+#DS- Let's copy+paste some code from the data onboarding document to get a general description of project. Furthermore, let's also specify what goals of the project this repository seeks to achieve.
```
Comment (py5gol) [inline README.md:3]: Lets add it later.

File: README.md [hunk 6]
```diff
@@ -9,10 +10,11 @@ TODO: Guide users through getting your code up and running on their own system.
 4.	API references
 
 # Build and Test
-TODO: Describe and show how to build your code and run the tests. 
+TODO: Describe and show how to build your code and run the tests.
+#DS- We agreed on using Python, but we should also specify what packages/frameworks we will also be using. If necessary, let's also specify an IDE program that we could all use in order to have a consistent work environment.
```
Comment (py5gol) [inline README.md:14]: Packages go into the requirements.txt, and no need to specific IDE.

**Review by py5gol — CHANGES_REQUESTED** [review]

**Pull request closed without merging**